### PR TITLE
Add elementary Code

### LINKED ISSUE #86: Add elementary Code
We're [working on EditorConfig support](https://github.com/elementary/code/issues/323) in [elementary Code](https://github.com/elementary/code)! We'd love to be added once [the PR](https://github.com/elementary/code/pull/345) is merged.

Icon:
icon

### PR DESCRIPTION
**Pull request by Pittan**
Closes #86

### CODE REVIEW

**Comment by Pittan** [comment]
@treyhunner Please review 🙇 
@cassidyjames Is this all right? 🤔

**Comment by cassidyjames** [comment]
@Pittan works for us! Thank you so much!

**Comment by Pittan** [comment]
@xuhdev Please review and merge 🙇

File: index.html [hunk 1]
```diff
@@ -125,6 +125,7 @@ <h2>No Plugin Necessary</h2>
   <ul class="editor-logos">
     <li><a href="http://barebones.com/support/technotes/editorconfig.html"><img src="logos/bbedit.png" alt="BBEdit"><span>BBEdit</span></a></li>
     <li><a href="https://github.com/eranif/codelite/tree/master/EditorConfigPlugin"><img src="logos/codelite.png" alt="CodeLite"><span>CodeLite</span></a></li>
+    <li><a href="https://github.com/elementary/code"><img src="logos/elementary-code.png" alt="elementary Code"><span>elementary Code</span></a></li>
```
Comment (xuhdev) [inline index.html]: Does it have an individual page? I couldn't find it but it seems like there should be one. If not, can you change the link to `https://github.com/elementary/code#readme`?

Comment (Pittan) [reply index.html]: @cassidyjames Does it have an individual page?
@xuhdev I changed the link. If it doesn't have a page, please merge it 🙇

**Comment by cassidyjames** [comment]
@Pittan the editorconfig implementation is available at https://github.com/elementary/code/tree/master/plugins/editorconfig, but there's not a specific landing page for it or anything.

**Comment by xuhdev** [comment]
Cool, thanks! :fireworks:

**Pull request merged**